### PR TITLE
[MFTF] Refactoring of AdminUpdateSimpleProductWithRegularPriceInStockEnabledFlatTest

### LINKED ISSUE #31234: [Issue] [MFTF] Refactoring of AdminUpdateSimpleProductWithRegularPriceInStockEnabledFlatTest
This issue is automatically created based on existing pull request: magento/magento2#31221: [MFTF] Refactoring of AdminUpdateSimpleProductWithRegularPriceInStockEnabledFlatTest

---------
### Description (*)
The test is refactored according to the best practices

### Manual testing scenarios (*)
1. Enable flat catalog product
2. Create a simple product
3. Update product (main info) and save
4. Check if all changes are available on the Product Edit page in Admin
5. Check if all changes are available on the Product Detail page on Storefront
6. Check if search by the new SKU is successful

### PR DESCRIPTION
**Pull request by AnnaAPak**
### Description (*)
The test is refactored according to the best practices

### Manual testing scenarios (*)
1. Enable flat catalog product
2. Create a simple product
3. Update product (main info) and save
4. Check if all changes are available on the Product Edit page in Admin
5. Check if all changes are available on the Product Detail page on Storefront
6. Check if search by the new SKU is successful

### Resolved issues:
1. [x] resolves magento/magento2#31234: [MFTF] Refactoring of AdminUpdateSimpleProductWithRegularPriceInStockEnabledFlatTest

### CODE REVIEW

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminAssertManageStockOnEditPageActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertManageStockOnEditPageActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminAssertManageStockOnEditPageActionGroup.xml]: Please, make sure you are following the guidelines for naming. See 11.3.5.1 item of https://devdocs.magento.com/guides/v2.4/coding-standards/technical-guidelines.html#113-functional-testing

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminAssertProductInfoOnEditPageActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertProductInfoOnEditPageActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminAssertProductInfoOnEditPageActionGroup.xml]: Please, correct the naming by using the "Assert" prefix before specifying the area. Thank you

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminAssertProductIsAssignedToCategoryActionGroup.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminAssertProductIsAssignedToCategoryActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminAssertProductIsAssignedToCategoryActionGroup.xml]: Please, correct the naming by using the "Assert" prefix before specifying the area. Thank you

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductStockStatusOnProductPageActionGroup.xml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontAssertProductStockStatusOnProductPageActionGroup">
```
Comment (rogyar) [inline app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductStockStatusOnProductPageActionGroup.xml]: Please, correct the naming by using the "Assert" prefix before specifying the area. Thank you

**Review by rogyar — CHANGES_REQUESTED** [review]
Hi @AnnaAPak. Good job. Please, check my comments below

**Comment by sidolov** [comment]
@magento create issue

**Comment by AnnaAPak** [comment]
@magento run Functional Tests CE, Functional Tests EE, Functional Tests B2B

File: app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductStockStatusOnProductPageActionGroup.xml (hunk 4 above)
Comment (AnnaAPak) [reply app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductStockStatusOnProductPageActionGroup.xml]: @rogyar  I have fixed naming for all Action Groups. Can you please take a look one more time? Thanks.

**Review by rogyar — APPROVED** [review]

**Comment by engcom-Alfa** [comment]
QA not applicable

**Comment by engcom-Hotel** [comment]
@magento run all tests

**Pull request merged**